.storybook/webpack.config.js

```diff
@@ -19,6 +21,14 @@ module.exports = ({ config }) => {
     test: /\.(ts|tsx)$/,
     use: [require.resolve("babel-loader"), require.resolve("react-docgen-typescript-loader")]
   });
+
+  // loader for scss files
+  config.module.rules.push({
```

Comment:
can we not use sass ?
- why do we need sass ?
- node-sass is pretty annoying (it's compiled on every machine ..)